Silicon/AlderlakePkg/Include/MkhiMsgsAdl.h

```diff
@@ -60,28 +60,6 @@
 
 #define ARB_SVN_COMMIT_ALL                      0xFF ///< Value 0xFF refers to committing all pending ARBSVN values.
 
-//
```

Comment:
What is the reason for moving these API's to common file?
I would recommend to maintain in ADL. We can review and move these definitions to common later,

Reply:
There is MKHI_MESSAGE_HEADER dependencies in common MkhiMsgs header file which causing duplicate definition issue if we merge into Adl specific header file. Currently only the header definition is in common header but the API still in ADL specific Heci lib